doc/specs/vulkan/chapters/memory.txt

```diff
@@ -775,29 +772,29 @@ same memory object.
 When memory is allocated, its contents are undefined.
 
 There is an implementation-dependent maximum number of memory allocations
-which can: be simultaneously created on a device.
+that can: be simultaneously created on a device.
 This is specified by the
 <<features-limits-maxMemoryAllocationCount,pname:maxMemoryAllocationCount>>
 member of the sname:VkPhysicalDeviceLimits structure.
 If pname:maxMemoryAllocationCount is exceeded, fname:vkAllocateMemory will
 return ename:VK_ERROR_TOO_MANY_OBJECTS.
 
-[NOTE]
-.Note
-====
 Some platforms may: have a limit on the maximum size of a single allocation.
 For example, certain systems may: fail to create allocations with a size
 greater than or equal to 4GB.
 Such a limit is implementation-dependent, and if such a failure occurs then
-the error ename:VK_ERROR_OUT_OF_DEVICE_MEMORY should: be returned.
-====
+the error ename:VK_ERROR_OUT_OF_DEVICE_MEMORY must: be returned.
 
 .Valid Usage
 ****
-  * [[VUID-vkAllocateMemory-device-00636]]
-    The number of currently valid memory objects, allocated from
-    pname:device, must: be less than
-    sname:VkPhysicalDeviceLimits::pname:maxMemoryAllocationCount
+  * pname:pAllocateInfo::pname:allocationSize must: be less than or equal to
+    sname:VkPhysicalDeviceMemoryProperties::pname:memoryHeaps[pname:pAllocateInfo::pname:memoryTypeIndex].pname:size
```

Comment:
nit: Should be `slink:`

Reply:
IMO `*link` should be everywhere. So useful.

Reply:
Yea the style guide *should* actually say that now - but people still miss it. The only time you don't use it now is when you're within the thing you'd be linking to (i.e. the description of VkPhysicalDeviceProperties would have "sname:VkPhysicalDeviceProperties".)